dockerfiles/README.md

```diff
@@ -46,11 +46,24 @@ without Docker Compose, you can try the following command before loading your
 vocabulary data. It starts a container in the same way our other example with
 the `docker-compose` command.
 
+    export JENA_4_VERSION=4.7.0
+
+    docker build -t jena-fuseki:$JENA_4_VERSION --build-arg JENA_VERSION=$JENA_4_VERSION \
+        --no-cache dockerfiles/jena-fuseki2-docker
+
+    mkdir -p databases/skosmos
+    # You need to install an Apache Jena Binary Distribution version
+    # from https://jena.apache.org/download/index.cgi
+    tdb2.tdbloader --loc databases/skosmos ./dockerfiles/config/skosmos.ttl
```

Comment:
`stain/jena-fuseki` has some features that Jena's Dockerfile is missing like configuring security with Apache Shiro, the UI, and also automatically running `tdbloader` inside the image.

For the Apache Jena image we need to ask users to run that before starting the container, to prepare the database/dataset to be used.

Reply:
I think it would be better to advise users to use s-put or curl to upload the RDF data from outside the container. That way Fuseki would be creating and controlling the TDB from the outset. Later the data can be updated using the same method.

I'm not against preloading the dataset using tdbloader as an option - it might even be necessary for very large data sets (though Skosmos might struggle with those anyway). But I don't think it's ideal to ask users to install Jena tools just for the setup.

Reply:
I used `tdbloader` as it's in the Jena Docker docs - https://github.com/apache/jena/tree/main/jena-fuseki2/jena-fuseki-docker

But I will test and then change to `curl` (will try this one first as it's used in the previous example with just `docker`).